Assets/Scripts/Networking/ConnectionHandler.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ConnectionHandler : Photon.PunBehaviour {
+
+	[SerializeField] string gameVersion = "0";
+
+	public List<Selectable> connectionReqdObject = new List<Selectable>();
+	List<bool> croStartingStates = new List<bool>();
+
+	bool isInRoom = false, startMatch = false;
```

Comment:
```csharp
bool isInRoom = false;
bool startMatch = false;
```

Reply:
bah, I'd care if they weren't closely tied to each other

Reply:
This is a readability concern for me, if they're closely tied to each other they can still live next to each other when declared, but a casual observer might miss that two variables are being declared on this line. IMO it's good practice to keep one declaration / line